policyengine_us/parameters/gov/states/mi/tax/income/deductions/retirement_benefits/tier_three/amount.yaml

```
@@ -0,0 +1,30 @@
+description: Michigan provides this retirement and pension benefits amount if you or your spouse are of qualifying age.
+metadata:
+  label: Michigan retirement and pension benefits for born 01/01/1956 through 01/01/1961 and at the age of 62 to 66
```

Avoid including parameter values in labels and descriptions
```suggestion
  label: Michigan retirement and pension benefits deduction limit
```